src/lib/ext-monitor.js

```diff
@@ -20,14 +23,15 @@ export default class ExtensionMonitor {
     const tab = await browser.tabs.query({
       url: getActivityLogPageURL(),
     });
+
     return tab.length > 0;
   }
 
   // If the Activity Log page is open, each logs will be send to
   // Activity Log page (as soon as it is encountered).
   async sendLogs(details) {
     const isExtPageOpen = await this.isActivityLogPageOpen();
-    if (isExtPageOpen) {
+    if (isExtPageOpen || this.devToolsPanelTabIds.size) {
```

Comment:
If there are any active ActivityLog devtools panel, the realtime logs will be sent.